src/test/ui/async-await/async-fn-size.rs

```diff
@@ -1,9 +1,10 @@
+// run-pass
+
 // edition:2018
 
 #![feature(async_await, await_macro)]
 
-#[path = "../auxiliary/arc_wake.rs"]
-mod arc_wake;
+extern crate arc_wake;
```

Comment:
There's an auxiliary file in `src/test/run-pass/auxiliary/arc_wake.rs` still here. Could you either move/remove it depending on whether there's any difference between it and the one inside `src/test/ui/async-await/auxiliary/arc_wake`?

Reply:
Thanks for your review. I'll remove the duplicate. BTW what's the purpose of auxiliary files?

Reply:
Hi @Centril The test `run-pass/futures-api.rs` also uses `run-pass/auxiliary/arc_wake.rs`. Shall I move `futures-api.rs` to `ui`?

Reply:
@agnxy Yes that seems like a good idea to have them co-located, that is `src/test/ui/async-await/futures-api.rs`.

Reply:
FYI I asked @cramertj about this and they pushed back on grouping them together: https://github.com/rust-lang/rust/pull/61818#issuecomment-505234220

That's why I ended up using the `#[path]` attribute.

Reply:
I see I agreed with @cramertj back then, but in hindsight it's better not to duplicate the auxiliary file...
I would also like not to have bare ungrouped files right in side `src/test/ui/` (we have lots of them today but you have to start somewhere).

Reply:
> in hindsight it's better not to duplicate the auxiliary file

Yes; that's why I used `#[path = "../auxiliary/arc_wake.rs"]` in some places rather than the `aux-build` directive.

> I would also like not to have bare ungrouped files right in side `src/test/ui/`

Agreed, in principle. Does that mean we should still have a group if it only fits one test, or try to find groups that are a less-than-perfect fit for a particular test? Personally, I'm fine either way.

Reply:
> Does that mean we should still have a group if it only fits one test, or try to find groups that are a less-than-perfect fit for a particular test?

I think you have to do that since grouping is always an act which forgets about some details for some bigger picture. You can always have groups in groups.